src/pages/dogs/[id]/index.jsx

```diff
@@ -466,6 +480,8 @@ export default function IndividualDogPage() {
               ) : (
                 <div className="grow flex gap-4 justify-end">
                   <div className="flex gap-4">
+
+                  {(userRole === "Admin" || userRole === "Manager" || userRole === "Instructor/Caregiver") && 
```

Comment:
I don't think there is a role for "Instructor/Caregiver" unless then endpoint was changed. What you can do is check if `dog.association != "Volunteer/Partner"`. If that is the case, we know the user must be a Admin, Manager, or have association Instructor/Caregiver